src/main/scala/cromwell/engine/backend/jes/Pipeline.scala

```diff
@@ -28,7 +28,7 @@ object Pipeline {
       callTag = Option(key.tag)
     )
 
-    logger.debug(s"Command line is: $command")
+    logger.info(s"Command line is: $command")
```

Comment:
Oh right... this should be changed back to debug.  I changed it to info because I wanted to see the contents of that log message.  Though it is kind of funny that "debug" is basically the equivalent of /dev/null because of how polluted it is